src/client/Client.cc

```diff
@@ -5040,8 +5040,26 @@ static bool has_new_snaps(const SnapContext& old_snapc,
   return !new_snapc.snaps.empty() && new_snapc.snaps[0] > old_snapc.seq;
 }
 
+struct SnapRealmInfoMeta {
+  utime_t last_modified;
+  uint64_t change_attr;
+};
```

Comment:
You may need to explicitly declare the constructors here.

Reply:
yeh, we may add complex types and its good to have the ctor.